src/data/nat/fib.lean

```diff
@@ -96,4 +97,74 @@ begin
     mono }
 end
 
-end nat
+/-- Subsequent Fibonacci numbers are coprime,
+  see https://proofwiki.org/wiki/Consecutive_Fibonacci_Numbers_are_Coprime -/
+lemma fib_succ_coprime (n : ℕ) : nat.coprime (fib n) (fib (n + 1)) :=
```

Comment:
Maybe `fib_coprime_fib_succ`?
I guess you can write the statement as `(fib n).coprime (fib (n + 1))` if you want.